aunt-leahs-app/src/components/header.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import SubText from './SubText.jsx';
+import constants from '../constants.jsx';
+import '../styles.css';
+
+const headerText = constants.HEADER.MAIN_HEADER;
+
+const Head = ({ page }) => {
+    /*takes in 'page' as props. Refer to ./components/constants.jsx for props.page values */
+    return (
+        <div className='headerBox'>
+            <img className="headerImage header" alt="Aunt Leah's Logo" src={require('../AuntLeahsTrees.png')} />
+            <div className="headerMain header">{headerText.headTwo}</div>
+            {(page !== null)
+                && <SubText title={page} />}
```

Comment:
Personally, I'd move the `&&` up to line 14, but it doesn't really matter ¯\_(ツ)_/¯

Reply:
Since SubText is self-closing, this should be written as a single line.
```
{(page !== null) && <SubText title={page} />}
```
If the element to be rendered required more than one line, then the && operator should begin a new line.